rocky/katalogus/client.py

```diff
@@ -38,6 +38,7 @@ def dict(self, *args, **kwargs):
         plugin_dict["produces"] = {ooi_class.get_ooi_type() for ooi_class in plugin_dict["produces"]}
         return plugin_dict
 
+    @classmethod
```

Comment:
This should actually be a static method, not a class method, because it doesn't use the class object in the method. Changing it to staticmethod also means you need to remove the unused self argument from the def.

Reply:
I could not reproduce this, I think it was like a difference in versions of code